[2019-06-06 15:42:49,382] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: hello_world.hello_task 2019-06-05T12:00:00+00:00 [queued]>
[2019-06-06 15:42:49,386] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: hello_world.hello_task 2019-06-05T12:00:00+00:00 [queued]>
[2019-06-06 15:42:49,386] {__init__.py:1353} INFO - 
--------------------------------------------------------------------------------
[2019-06-06 15:42:49,386] {__init__.py:1354} INFO - Starting attempt 1 of 1
[2019-06-06 15:42:49,386] {__init__.py:1355} INFO - 
--------------------------------------------------------------------------------
[2019-06-06 15:42:49,508] {__init__.py:1374} INFO - Executing <Task(PythonOperator): hello_task> on 2019-06-05T12:00:00+00:00
[2019-06-06 15:42:49,508] {base_task_runner.py:119} INFO - Running: [u'airflow', u'run', 'hello_world', 'hello_task', '2019-06-05T12:00:00+00:00', u'--job_id', '35', u'--raw', u'-sd', u'DAGS_FOLDER//home/micah/Desktop/Work/ETL/hello_world.py', u'--cfg_path', '/tmp/tmp5jPiUB']
[2019-06-06 15:42:49,726] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task /home/micah/Desktop/Work/ETL/airflow_virtualenv/local/lib/python2.7/site-packages/airflow/configuration.py:590: DeprecationWarning: You have two airflow.cfg files: /home/micah/airflow/airflow.cfg and airflow_home/airflow.cfg. Airflow used to look at ~/airflow/airflow.cfg, even when AIRFLOW_HOME was set to a different value. Airflow will now only read airflow_home/airflow.cfg, and you should remove the other file
[2019-06-06 15:42:49,726] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task   category=DeprecationWarning,
[2019-06-06 15:42:50,124] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task [2019-06-06 15:42:50,124] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-06-06 15:42:50,399] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task [2019-06-06 15:42:50,399] {__init__.py:305} INFO - Filling up the DagBag from /home/micah/Desktop/Work/ETL/airflow_home/dags/home/micah/Desktop/Work/ETL/hello_world.py
[2019-06-06 15:42:50,446] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task Traceback (most recent call last):
[2019-06-06 15:42:50,446] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task   File "/home/micah/Desktop/Work/ETL/airflow_virtualenv/bin/airflow", line 32, in <module>
[2019-06-06 15:42:50,446] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task     args.func(args)
[2019-06-06 15:42:50,446] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task   File "/home/micah/Desktop/Work/ETL/airflow_virtualenv/local/lib/python2.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-06-06 15:42:50,446] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task     return f(*args, **kwargs)
[2019-06-06 15:42:50,446] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task   File "/home/micah/Desktop/Work/ETL/airflow_virtualenv/local/lib/python2.7/site-packages/airflow/bin/cli.py", line 501, in run
[2019-06-06 15:42:50,446] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task     dag = get_dag(args)
[2019-06-06 15:42:50,447] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task   File "/home/micah/Desktop/Work/ETL/airflow_virtualenv/local/lib/python2.7/site-packages/airflow/bin/cli.py", line 145, in get_dag
[2019-06-06 15:42:50,447] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task     'parse.'.format(args.dag_id))
[2019-06-06 15:42:50,447] {base_task_runner.py:101} INFO - Job 35: Subtask hello_task airflow.exceptions.AirflowException: dag_id could not be found: hello_world. Either the dag did not exist or it failed to parse.
[2019-06-06 15:42:54,412] {logging_mixin.py:95} INFO - [2019-06-06 15:42:54,412] {jobs.py:2562} INFO - Task exited with return code 1
